### ISSUE TITLE
Image metadata

### ISSUE THREAD

**Issue by pgyogesh** [ramswaroop/photography#16]
The metadata of image doesn't display on the first click but it displays on the second click.

Repro:
Just open the same image twice by clicking on the thumbnail. After second click you will see metadata.

**Comment by ramswaroop** [comment]
Yes, I am actually aware of this issue. The issue is due to the theme used. Technically, if we can pre-load the image then this issue won't occur but unfortunately, the pre-load image feature of the theme doesn't work.

**Comment by songyaojiang** [comment]
Same issue here. Anyone has solution to this problem?
